src/main/java/org/reactome/release/qa/check/HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck.java

```diff
@@ -7,21 +7,26 @@
 import org.reactome.release.qa.common.QACheckerHelper;
 import org.reactome.release.qa.common.QAReport;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Flags all human Reactions that contain non-disease PhysicalEntities that are non-human, or are human PhysicalEntities with relatedSpecies.
  */
 
 @SliceQATest
 public class HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck extends AbstractQACheck {
 
+    // Rather than overload multiple methods that use/don't use a skiplist, an empty List is used here.
+    private static List<String> EMPTY_SKIP_LIST = new ArrayList<>();
```

Comment:
Same here with removing `static`

Reply:
If `EMPTY_SKIP_LIST` is really going to be used instead of a few simple overloads, then it probably should not be duplicated in `HumanReactionsWithNonHumanComplexesWithHumanComponentsCheck` as well as `HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck`. Both classes extend `AbstractQACheck` so `EMPTY_SKIP_LIST` should be moved to that, or if it is not appropriate for _all_ `AbstractQACheck`s to have access to `EMPTY_SKIP_LIST`, then a new class or interface could be introduced, say, `HasNoSkipList` and then `HasNoSkipList` could extend `AbstractQACheck` and then `HumanReactionsWithNonHuman...Check` classes could extend `HasNoSkipList`.

Reply:
@SolomonShorser-OICR I was thinking that, too.  I like that approach but maybe it's best to wait for the next PR?  I get the sense there are existing checks outside the scope of this PR that will benefit from that kind of class or interface structure as well.

Reply:
I moved EMPTY_SKIP_LIST to `AbstractQACheck`